docs/cli/cartesi-rollups-cli_execute.md

```diff
@@ -26,6 +26,12 @@ cartesi-rollups-cli execute --voucher-index 5 --input-index 6
       --voucher-index int         index of the voucher
 ```
 
+### Options inherited from parent commands
+
+```
+  -v, --verbose   verbose output
+```
+
```

Comment:
Shouldn't we simply add the new option to the list above? This applies to all commands that inherited it.

Reply:
Thesed are all autogenerated files